webpack.config.js

```diff
@@ -18,17 +18,17 @@ const config = {
     loaders: [
       {
         test: /\.scss$/,
-        loader : 'style!css!sass!postcss'
+        loader : 'style-loader!css-loader!sass-loader!postcss-loader'
```

Comment:
Now that we're switching to webpack 2, let's switch this config to the webpack 2 specific way [using `use` instead of `loader`](https://webpack.js.org/guides/migrating/#module-loaders-is-now-module-rules).